pkgs/games/factorio/default.nix

```diff
@@ -1,5 +1,5 @@
 { stdenv, fetchurl, makeWrapper
-, alsaLib, libX11, libXcursor, libXinerama, libXrandr, libXi, libGL
+, alsaLib, libX11, libXcursor, libXinerama, libXrandr, libXi, libGL, libpulseaudio
```

Comment:
If Pulseaudio is now required, then alsaLib probably will not be. Add a TODO by stable to test that theory once it catches up with 0.17.68.